src/main/java/com/jcabi/aspects/aj/MethodCacher.java

```diff
@@ -390,6 +397,15 @@ public boolean expired() {
                 this.level = Loggable.DEBUG;
             }
         }
+
+        /**
+         * Get log level.
+         * @return Log level of current method.
+         */
+        public int getLevel() {
```

Comment:
@pinaf checkstyle hinted that final class should not contain protected methods when I used "protected".